packages/core/demo/data/index.ts

```diff
@@ -605,8 +607,9 @@ export const storybookDemoGroups = Tools.clone(allDemoGroups);
 
 // in the demo page we want to show only demos with isDemoExample = true
 export const demoGroups = Tools.clone(allDemoGroups)
+	// remove demoGroup if it's children don't have any demo examples
+	.filter((demoGroup) => demoGroup.demos.filter(demo => demo.isDemoExample !== false).length > 0)
 	.map((demoGroup) => {
-		demoGroup.demos = demoGroup.demos.filter(demo => demo.isDemoExample);
+		demoGroup.demos = demoGroup.demos.filter(demo => demo.isDemoExample !== false);
```

Comment:
Can this just be
```suggestion
		demoGroup.demos = demoGroup.demos.filter(demo => !demo.isDemoExample);
```

Reply:
when `isDemoExample` is `null` or `undefined` we don't want this condition to pass, and also it's safer type-wise this way